resources/views/ItemEdit/ItemEdit.blade.php

```diff
@@ -0,0 +1,304 @@
+    }
+    .input{
+        background-color:transparent; 
+        border-color:black; 
+        height:50px;
+        margin-top: 30px;
+    }
+    .close-button{
+        color:white;
+        background-color: rgb(102,153,204);
+        border-radius: 10px;
+        padding: 7px 32px;
+        text-align: center;
+        text-decoration: none;
+        font-size: 16px;
+    }
+    .save-button,.delete-button{
+        color:white;
+        background-color: rgb(102,153,204);
+        border-radius: 10px;
+        padding: 7px 32px;
+        text-align: center;
+        text-decoration: none;
+        font-size: 16px;
+    }
+    select{
+        background-color:transparent; 
+        border-color:black; 
+        height:50px;
+        margin-top: 30px;
+        text-align: center;
+    }
+
+    .sort{
+        background-color: rgb(235,245,251);
+        border-color: rgb(204,204,255);
+        border-radius: 10px;
+        margin-top: auto;
+        padding: 15px
+    }  
+    .inventory-text{
+        padding-left: 105px;
+    }
+    .inventory{
+        margin-top: 50px;
+    }
+    table{
+        background-color: rgb(51,102,153);
+        border-radius: 10px;
+        margin-bottom: 30px;
+    }
+    th ,td{
+        color: white;
+        width: 40px;
+        padding-left: 40px;
+        padding-top: 20px;
+    }
+    .nav{
+        padding-left: 570px;
+    }
+    .text-margin{
+        margin-left: 50px;
+    }
+    </style>
+
+    <div class="container">
+        <div class="row justify-content-center">
+
+            {{-- to edit item details --}}
+            <form action="#" method="post" enctype="multipart/form-data" id="form1">
+                @csrf
+                @method('PATCH')
+
+            </form>
+
+            <form action="#" method="post" id="form2">
+                @csrf
+                @method('DELETE')
+            </form>
+
+                <h1 class="text-center">EDIT ITEMS</h1>
+
+
+                <div class="row">
+                    {{-- Item card --}}
+                    <div class="col-4">
+                        <div class="row">
+                            <div class="col-10 align-items-center">
+                                <div class="card card-potision">
+                                    <div class="card-body">
+                                        <div class="upload-box">
+                                            <input type="file" name="upload_file" id="input_files" >
+                                            <span>+</span>
+                                        </div>                                
+                                    </div>
+                                    <div class="card-footer" style="background-color: rgb(216,230,233)">
+                                        <div class="border border-rounded" style="background-color: rgb(153,204,255)">
+                                            <p class="h2 text-center">ITEM NAME</p>
```

Comment:
Is this only temporary? Just static data

Reply:
Yes . this is front end part 
Is there something wrong?

Reply:
Yes . This is just front end code.
If it will go to back end part, I change this code.